src/Dashboard/dashboard.jsx

```diff
@@ -40,41 +41,46 @@ export function Dashboard({
 
   if (isAuthenticated) {
     return (
-      <div className="container Dashboard">
-        <div className="row align-items-center">
-          <div className="col-12 col-md-6 align-self-center">
-            <h2 className="welcomeUser light">Overview</h2>
-          </div>
-          <div className="col-auto">
-            <Link className="uploadBtn btn btn-primary" to="/upload">Upload Data</Link>
-          </div>
-          <div className="col-auto">
-            <Link className="downloadBtn btn btn-primary" to="/download">Download/View Data</Link>
-          </div>
-          {featureAvailable.dashboardEditable ? editBtn : ''}
-        </div>
-        <div className="row">
-          <div className="col">
-            <h3 className="divHeader">
-              {`${userDisplayName}, ${siteName}`}
-            </h3>
-          </div>
-        </div>
+      <div className="container-fluid">
         <div className="row">
-          { disconnectComponents ? <PreviousUploadsTable previousUploads={previousUploads} /> : <PreviousUploadsTableConnected />}
-          <PreviousUploadsGraph previousUploads={previousUploads} />
-        </div>
-        <div className="row">
-          <div className="col">
-            <h3 className="divHeader">
-              All Sites
-            </h3>
+          <Sidebar />
```

Comment:
I believe like the Sidebar should be implemented at the app.jsx level. In the current user flow, it is an essential element to always be present when a user is logged in.

Placing it here eliminates the ability of logged-in users to navigate back to the dashboard or any other area if they go to pages like `about us` or `contact`. This also increases our probability of forgetting to include the side-bar on a future page for logged in users. 

Let me know what you think about placing it in app.jsx, appearing if a user is logged in, instead of on individual pages.

Reply:
Per our discussion, I am open to implement the `sidebar` at the `app.jsx` level and make it available to the `team` and `external-links` pages, while making the `dashboard` page to be the _landing_ page for authenticated users.